framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/CliService.cs

```diff
@@ -63,6 +67,68 @@ public async Task RunAsync(string[] args)
             }
         }
 
+        private void CheckDependencies()
+        {
+            var installedNpmPackages = GetInstalledNpmPackages();
+
+            if (!installedNpmPackages.Contains(" yarn@"))
+            {
+                InstallYarn();
+            }
+            if (!installedNpmPackages.Contains(" gulp@"))
+            {
+                InstallGulp();
+            }
+        }
+
+        private string GetInstalledNpmPackages()
+        {
+            var output = "";
+
+            using (var process = new Process())
+            {
+                process.StartInfo = new ProcessStartInfo("CMD.exe")
```

Comment:
CMD.exe can not work on a non-Windows OS. Also fix CmdHelper if you find a solution.